tests/expr10.py

```diff
@@ -7,6 +7,8 @@ def test_UnaryOp():
     b = not 5
     b = not -1
     b = not 0
+    b = True
+    a = -b
```

Comment:
I think changes also include integer, real and complex types? Can we add tests for these types as well and try to cover as many kinds of each of these types as possible?